### ISSUE TITLE
The same block conversions if submitted on minute 0-1

### ISSUE THREAD

**Issue by ilzheev** [pegnet/pegnet#339]
Miners parse rates at minute 1 of each dblock.
We don't want people to trading knowing future, so we apply conversions with 1 block delay.

**Proposal:**
If conversion submitted on minute 0 or minute 1 (the specific minute of entry submission is accessible via factomd API), miners calculate such proposal in the next block (without delay).

This significantly improves arbitrage – basically you can "fix" conversion rate of minute 0-1 by submitting conversion and get this conversion rate in the next block.

**Comment by MitchellBerry** [comment]
This perhaps also belongs in Pegnet Improvement Proposals: https://docs.google.com/document/d/1-cDQ3KxFQjz7ydgVHrdXZjj9hT980ovcZL4_eT88XAg/edit?usp=sharing

**Comment by MitchellBerry** [comment]
I do not agree with proposals being on google docs, but this is the current place for such things.

**Comment by Emyrk** [comment]
@MitchellBerry all the ones that the devs *cough me* acknowledge are here <https://github.com/pegnet/pips>
